athlinks-schema.ttl

```diff
@@ -158,7 +195,7 @@ athlinks:pace
 athlinks:seriesType
   a owl:DataTypeProperty;
   rdfs:range [ a xsd:string ;
-                owl:oneOf("Master" "Social" "Points" "Series")
+                owl:oneOf("Master" "Social" "Points" "Series" "MasterSeries")
```

Comment:
It seems ambiguous to have data such as:
```turtle
foo athlinks:seriesType "Master".
bar athlinks:seriesType "MasterSeries".
```
What is the difference between these? Why do we need both?